source/about/zero-trust.rst

```diff
@@ -0,0 +1,127 @@
+
+Identity and access management
+------------------------------
+
+Centralized and secure identity management is a cornerstone of Zero Trust. Mattermost integrates seamlessly with enterprise identity providers (IdPs), enabling strong identity verification and strict access control.  
+
+By using one of the secure identity mechanisms listed below and enforcing least-privilege access via roles and groups, Mattermost ensures that only verified individuals gain access to the platform and its resources.  
+
+- `SAML <https://docs.mattermost.com/onboard/sso-saml.html>`_: Enables seamless Single Sign-On, ensuring secure, centralized authentication to verify every user's identity.
+- `LDAP <https://docs.mattermost.com/onboard/ad-ldap.html>`_: Facilitates integration with enterprise directories to tightly control user access, adhering to granular identity verification.
+- `OpenID Connect <https://docs.mattermost.com/configure/authentication-configuration-settings.html#openid-connect>`_: Provides secure, standards-based user authentication to verify identities and enforce secure access.
+- `Entra ID OAuth 2.0 <https://docs.mattermost.com/configure/authentication-configuration-settings.html#entra-id-oauth-2-0-settings>`_: Supports access via verified identity providers, ensuring strict authentication and access control.
+- `Session Management <https://docs.mattermost.com/configure/environment-configuration-settings.html#session-lengths>`_: Enhances security by limiting session lifetimes and enforcing strict session policies, reducing the risk of unauthorized access.
+
+Authorized users can seamlessly be added and removed from channels utilizing the native AD/LDAP integration based on group memberships:  
+
+- `LDAP Synchronized User Groups <https://docs.mattermost.com/onboard/ad-ldap-groups-synchronization.html>`_: Automates user management and access control by dynamically syncing with organizational directories to minimize risks and enforce policies.
+
+Micro-segmentation
+-------------------
+
+Segmenting and isolating sensitive resources is vital in minimizing lateral movement during an attack. Mattermost supports micro-segmentation through its organizational and role-based capabilities. Micro-segmentation enables organizations to restrict access to sensitive conversations, ensuring secure communication channels tailored to individual teams or missions.  
```

Comment:
Q: Is this section intentionally without doc references?

Reply:
Yes - docs references are included in subsequent sub-sections that all roll up to micro-segmentation. Please advise if you'd like this information structured in a different way, or want to highlight specific docs pages in the parent section.